content/docs/command-reference/add.md

```diff
@@ -83,9 +78,10 @@ in the directory tree. Instead, the single `.dvc` file references a special JSON
 file in the cache (with `.dir` extension), that in turn points to the added
 files.
 
-Note that DVC commands that use tracked files support granular targeting of
-files, even when the directory is added as a whole. Examples: `dvc push`,
-`dvc pull`, `dvc get`, `dvc import`, etc.
+Note that DVC commands that use tracked files or directories support targeting
```

Comment:
Even though, directory added "as a whole" and only single `.dvc` file is created, individual files in this directory can be accessed separately. For example, `dvc pull directory/path/to/file` would download a single file instead of the whole directory, same for `dvc push`, `dvc checkout`, etc.

Reply:
link to the example probably not that relevant on this case?

Reply:
Updated in 99a97899. And true, link removed. Other similar notes should probably link to this section too, instead of the example...